relay/kusama/constants/src/lib.rs

```diff
@@ -118,6 +118,18 @@ pub mod system_parachain {
 
 	// System parachains from Kusama point of view.
 	pub type SystemParachains = IsChildSystemParachain<Id>;
+
+	/// Coretime constants
+	// FAIL-CI @donal please check/fix
```

Comment:
@seadanda please check all the @.

Reply:
We need to make the `fast-runtime` timeslice at least 20 blocks long; otherwise, it doesn't make sense in the context of automated tests. (Okay, maybe 16 would suffice, but I'm for round decimal numbers in this case).

Reply:
Also, the package does not currently expose the feature, nor is it propagated from downstream packages.

Reply:
Ah @s0me0ne-unkn0wn you beat me to it 😀

Will review your PR

Reply:
@ggwpez all the other @s look fine and should not fail after https://github.com/polkadot-fellows/runtimes/pull/384